src/Widgets/AgendaEventRow.vala

```diff
@@ -291,6 +292,7 @@ public class Maya.View.AgendaEventRow : Gtk.ListBoxRow {
 
         is_allday = Util.is_all_day (start_date, end_date);
         is_multiday = Util.is_multiday_event (ical_event);
+        var is_same_time = start_date.equals (end_date);
```

Comment:
Looks like we accidentally added an `s` here

```suggestion
        var is_same_time = start_date.equal (end_date);
```

Reply:
Damn it!